### DISCUSSION TITLE
Support for Azure DevOps Git repos

### DISCUSSION THREAD

**Discussion by pasasorin** [fluxcd/flux2#667] [Q&A] [answered]
Hello,

from my understanding, there is no support for Azure DevOps Git repos. 
But setting that aside, are there really technical limitations if using Azure DevOps Git repos as source of truth for Flux?

Thank you!

**Comment by phillebaba** [comment, marked as answer]
Support for Azure DevOps has been added in the latest release. You will need to enable an extra feature to enable it.

More information can be found on the docs page.
https://toolkit.fluxcd.io/guides/installation/#generic-git-server
